tests/conftest.py

```diff
@@ -6,10 +6,7 @@
 import pytest
 
 
+# No longer needed, this also works with Trio
 @pytest.fixture
 def anyio_backend() -> tuple[str, dict[str, Any]]:
-    if sys.platform == "win32":
-        from asyncio.windows_events import WindowsSelectorEventLoopPolicy
-
-        return ("asyncio", {"policy": WindowsSelectorEventLoopPolicy()})
-    return ("asyncio", {})
+    return ("trio", {})
```

Comment:
This means that the test suite will _only_ run against Trio now. The proper fix is to delete the fixture completely.

Reply:
That's temporary until the next anyio release and/or I can figure out the reason for the test failure.

Reply:
Removed.